index.txt

```diff
@@ -60,11 +60,11 @@
 |                                     |  |                                     |
 | XORG                                |  | WAYLAND                             |
 |                                     |  |                                     |
-|  @/xorg/xorg-server                 |  |  TODO                               |
-|  @/xorg/x11-forwarding              |  |  TODO                               |
-|  @/xorg/sx                          |  |  TODO                               |
-|  @/xorg/keyboard-layout             |  |  TODO                               |
-|  @/xorg/x11vnc                      |  |                                     |
+|  @/xorg/xorg-server                 |  |  @/wayland/getting-started          |
```

Comment:
I think this section contains a lot of information. To break it up into more clear devided sections may help to understand. As for now the wayland implementation wlroots eg in the "enable xwayland" is mentioned. 
This somehow gives the assumption that wayland is wlroots.
I think something like:

@wayland/getting-started (Just mention how to make the base system ready for *wayland*, implementation agnostic)
@wayland/wlroots 
@wayland/swc
@wayland/kde

Reply:
I was hoping somebody would bring this up to confirm my suspicion. I love your suggestion, because it makes the pain-points of `wayland` more salient - that being the lack of uniformity across (if only theoretical) compositor implementations of the protocols `wayland` supports. 

This will probably be the direction I go with this, after re: `velox`. 

Getting a native `wayland` session in KDE would be marvelous and I should try that later as well. Hold my feet to the fire on this.

Reply:
Feet unburnt. 

Now that `zig` is in community, my goal is to have this section finished soon! I'll refactor it using your suggestions -- finally I will have made wiki entries that are small and palatable.